html/changelogs/archive/2021-07.yml

```diff
@@ -796,138 +796,41 @@
       can it be pushed (grab a dead core to swap places)
   - bugfix: borgs can now no longer be pushed when in combat mode and alive, you will
       always swap places with dead borg corpses.
-  PositiveEntropy, tf-4:
-  - imageadd: Thanks to a collaboration between the frontier sector and the core sector,
-      the parade jackets now boast new and varied apperances for all to enjoy, with
-      a new parade jacket releasing for the Head of Security!
-  - imageadd: In no short effort than the finest of tailors, the captain's hat, the
-      centcom hat, the captain's jumpsuit and the captain's carapace now have finer
-      gold trims and the finest quality leather available, making them more vibrant
-      for all!
-  - expansion: To utilize the new variants, simply alt-click a parade jacket, or a
-      centcom commander coat. You can also obtain a Head of Security parade jacket
-      by simply accessing the Head of Security's closet!
-  TheSmallBlue:
-  - expansion: Adds the pathfinding component.
-  Thunder12345:
-  - code_imp: Killed the miscellaneous items file!
   Wallem:
   - bugfix: Only foods which are created at mapload require player interaction in
       order to decompose.
-  Wayland-Smithy:
-  - spellcheck: Made the Doppler Array explosive sale message less wordy.
-  - bugfix: Fixed Chat Highlighting working on your own Binary messages.
-  dragomagol:
-  - bugfix: lavaland syndie base and seed vault no longer explode into green tiles
-  timothymtorres:
-  - expansion: Added xeno plasma statue
-  tralezab:
-  - spellcheck: Tiger Cooperative Traitors now have a slightly better uplink information
-      flavor text
-  - code_imp: killed the global vars misc file!
-2021-07-28:
-  Cyberboss:
-  - bugfix: Fixed APC's using extreme amounts of power instantly
-  EuSouAFazer:
-  - expansion: Kilostation Medbay's solar now starts with a Space Carp that cannot
-      destroy stuff.
-  - expansion: Kilostation Security's solar now doesn't have a solar computer, instead
-      having an empty computer frame and the circuitboard nearby. There is also 2
-      glass nearby as well.
-  - expansion: Kilostation Engineering's solar now starts partially built, with a
-      box with assembly and glass nearby.
-  Fikou:
-  - bugfix: the elite syndie suit now has the proper icons on its first equip
-  Guestify:
-  - bugfix: Replaces security newscasters with non security ones in non security rooms
-      on Tramstation
-  Kylerace:
-  - balance: tram is now twice as fast, pray it doesnt get faster
-  Wayland-Smithy:
-  - qol: Poll Candidates alerts no longer steal window focus, like ghost role rolls
-      and cult leadership votes.
-  - code_imp: Added an autofocus argument to tgui_alert, defaults on, that can be
-      used to prevent accidental inputs from surprise alerts when used.
-  ZephyrTFA:
-  - bugfix: Rust Heretics can once again benefit from rust
```

Comment:
there are literally zero code changes in this PR somehow

Reply:
I forgot to swap branches, and for some reason my changelog archive is conflicting because nothing can ever go smoothly.